packages/muelu/src/MueCentral/MueLu_Hierarchy_def.hpp

```diff
@@ -74,6 +74,28 @@
 
 #include "Teuchos_TimeMonitor.hpp"
 
+namespace {
+  template<class XpetraMatrixType>
+  Xpetra::global_size_t
+  getFakeGlobalNumEntries (const XpetraMatrixType& Am)
```

Comment:
Not sure I like the name for this @jhux2 ?

Reply:
I'm not crazy about this either.  See my comment about just skipping complexity if the global nnz(A) aren't available.

Reply:
I'm guessing the latter -- I think the point of this is just to make `describe()` work for Xpetra objects.  It might be better just to fix Xpetra so it doesn't print this information.

Reply:
Mark says that this function was his idea.  We could change the name.  The point was to avoid touching a lot of describe() implementations.